asm/macros/event.inc

```diff
@@ -1082,13 +1082,16 @@
 	@ Hides the secondary box spawned by showmoney.
 	.macro hidemoneybox
 	.byte 0x94
+	.byte 0
```

Comment:
Could add a comment that the game appends zeroes to this command(ruby's leftover)